packages/bot/src/transformers/invite.ts

```diff
@@ -42,6 +42,7 @@ export function transformInvite(
     if (props.expiresAt && payload.invite.expires_at) {
       invite.expiresAt = Date.parse(payload.invite.expires_at)
     }
+    if (props.flags && payload.invite.flags) invite.flags = new ToggleBitfield(payload.invite.flags)
```

Comment:
```suggestion
    if (props.flags && payload.invite.flags) invite.flags = payload.invite.flags
```
Took me a while to realize, but I think you're confusing between Flags and Toggles
- For Flags, the transformer keep it as-is (since it's just a number/bitfield). For example, https://github.com/discordeno/discordeno/blob/598b839af7b2473507d0a0833adb9aa0c2829c40/packages/bot/src/transformers/channel.ts#L107
- When there are multiple boolean keys returned from Discord, Toggles are constructed from ToggleBitfield. Toggles merge multiple boolean into one bitfield.

Reply:
My bad then, I guess the channel transformer is the one that needs to be updated.